{
    "functionName": "test_bad_df_columns",
    "className": "CleanDFOrSeriesTestCase",
    "fileName": "/mzy2240_&_ESA/tests_&_test_saw.py",
    "projectName": "repos",
    "Label": false,
    "isTest": true,
    "Body": "\"\"\"If the DataFrame columns are not valid fields, we should get\n        an error.\n        \"\"\"\nbad_df = pd.DataFrame([[1, 'bleh']], columns=['BusNum', 'bleh'])\nwith self.assertRaisesRegex(ValueError, 'The given object has fields'):\n    saw_14.clean_df_or_series(obj=bad_df, ObjectType='gen')\n"
}